### PR TITLE
Fix reverse-levels feedback duplication - DE46822

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
Visually, when reversing levels it appeared that feedback had been duplicated when it shouldn't. ( A refresh fixes it) The issue was that the parent's state was not being kept up-to-date with child changes so it was not triggering a re-render when it should have been. 

| Before | After | 
| --- | --- |
| reverse-before | reverse-after |

### CODE REVIEW

File: src/components/editor/rubric-feedback-editor.js [hunk 1]
```diff
@@ -47,12 +47,6 @@ class RubricFeedbackEditor extends composeMixins(
 		`;
 	}
 
-	constructor() {
-		super();
-		this.keyLinkRels = ['self'];
-		this.criterionName = '';
-	}
```
Comment (imcdonald-d2l) [inline src/components/editor/rubric-feedback-editor.js:54]: Un-related to this defect but we shouldn't be setting public properties in a constructor. I checked and these are set properly in the parent so this won't break anything.

Comment (mpharoah-d2l) [reply src/components/editor/rubric-feedback-editor.js:54]: It looks like these never change. Should they just be constants?

Comment (imcdonald-d2l) [reply src/components/editor/rubric-feedback-editor.js:54]: I'm confused by what you mean. These are public properties set by the parent. They shouldn't be assigned to constants. 
image

Comment (mpharoah-d2l) [reply src/components/editor/rubric-feedback-editor.js:54]: Oh, I was looking at the test thinking it was part of the actual component. I see now

**Review by mpharoah-d2l — APPROVED** [review]

**Pull request merged**